content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,82 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports will be captured by default.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics to the Citadel agent to monitor SDS push.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecars` resource.
+- **Added** the `ca.istio.io/env` and `ca.istio.io/override` labels for operators to control service account secret generation by namespace.
+- **Improved** the ingress gateway to load a CA certificate from a separate secret when using SDS.
```

Comment:
```suggestion
- **Added** endpoints `/debug/sds/workload` and `/debug/sds/gateway` on port 8080 of Citadel Agent for better debuggability.
- **Added** the CtrlZ endpoint on port 9876 of Citadel Agent for better debuggability.
- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor SDS service.
- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
```

Reply:
I revised and moved the first two items here.

Reply:
accepted.